setup.py

```diff
@@ -1,24 +1,14 @@
 from setuptools import setup
 
+install_requires =[]
```

Comment:
...but I think your `if r:` test is definitely more pythonic then my `if len(r)>0:`